src/sentry/web/frontend/restore_organization.py

```diff
@@ -54,29 +55,21 @@ def get(self, request, organization):
         return render_to_response('sentry/restore-organization.html', context, self.request)
 
     def post(self, request, organization):
-        if organization.status != OrganizationStatus.PENDING_DELETION:
+        deletion_statuses = [
```

Comment:
I'm assuming you're ok with this since you explicitly changed it, but I assumed once deletion actually began, you couldn't reverse. Since at that point, things can be in any number of broken states depending on how far the deletion has progressed. I don't think it's safe to allow this considering we already give a 24 hour window to cancel.

Reply:
Nevermind, you commented this in the PR.